pallets/automation-time/src/tests.rs

```diff
@@ -511,7 +511,7 @@ fn trigger_tasks_completes_some_tasks() {
 		);
 		LastTimeSlot::<Test>::put(LAST_BLOCK_TIME);
 
-		AutomationTime::trigger_tasks(90_000);
+		AutomationTime::trigger_tasks(80_000);
```

Comment:
i think the loop before checked for 60k, now is only checking for 10k, so this is why we need to lower the total weight, else everything runs